pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -900,19 +905,19 @@ func getLocationFromPod(clientset kubernetes.Interface, d *apps.Deployment, crus
 	return GetLocationWithNode(clientset, nodeName, crushRoot, hostName)
 }
 
-func GetLocationWithNode(clientset kubernetes.Interface, nodeName string, crushRoot, crushHostname string) (string, error) {
+func GetLocationWithNode(clientset kubernetes.Interface, nodeName string, crushRoot, crushHostname string) (string, string, error) {
```

Comment:
Named return values might help with understanding the function's use. Also a godoc.

Reply:
Godoc?

Reply:
Godoc?